Makefile

```diff
@@ -315,6 +315,18 @@ codespell:
 .PHONY: validate
 validate: lint .gitvalidation validate.completions man-page-check swagger-check tests-expect-exit pr-removes-fixed-skips
 
+.PHONY: validatepr
+validatepr:
+	$(PODMANCMD) run --rm --env HOME=/root \
+		-v $(CURDIR):/go/src/github.com/containers/podman \
+		--security-opt label=disable \
+		quay.io/baude/validatepr:latest  \
+		make .validatepr
+
+.PHONY: .validatepr
+.validatepr:
+	env BUILDTAGS="$(BUILDTAGS)" REMOTETAGS="$(REMOTETAGS)" bash -c "sh contrib/validatepr/validatepr.sh"
```

Comment:
why the double extra `bash -c sh ...` just make the script executable and call `contrib/validatepr/validatepr.sh` directly